README.md

```diff
@@ -1 +1,26 @@
-# project-h
\ No newline at end of file
+# Hello-Microverse
+In this project, we setup a  "Hello Microverse" project.
+
+## Build with
+- HTML
+- CSS
+
+## Live Demo
+Live demo link
+
+## Authors
+- GitHub: https://github.com/Vianney992
+- Twitter: https://twitter.com/AhononV
+
```

Comment:
- [ ] Great job documenting your project! It would be great to add your name in the Author section, so people can know how to call you.